.github/workflows/check-branch-name.yml

```diff
@@ -0,0 +1,21 @@
+name: Check pull request head branch name
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  check_branch_name:
+    if: github.head_ref == 'master' && github.repository == 'CleverRaven/Cataclysm-DDA'
+    uses: actions/github-script@v6.3.3
+    with:
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+      script: |
+        await github.rest.issues.createComment({
+          owner: context.repo.owner,
+          repo: context.repo.repo,
+          issue_number: context.issue.number,
+          body: "You are creating a merge request with the master branch as the head branch. This is likely a mistake unless you really know what you are doing. You may read https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on for a typical workflow of contributing to a project on GitHub."
```

Comment:
```suggestion
          body: "You are creating a pull request with the master branch as the head branch. This is likely a mistake unless you really know what you are doing. You may read https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on for a typical workflow of contributing to a project on GitHub."
```

Reply:
if I were to suggest something to someone new to github, I would try to not refer to one thing with, say, 2 different names; merge request may mean the same thing as pull request but keeping it simple for new contributors is nice